Author: George Kiagiadakis, Diane Trout
Description: Force a SO version number on libktpchat.
 later update so the SO version number defaults to the package version.
Update: Jonathan Riddell 2014-29-10 to match upstream and use ktp version number

Index: ktp-text-ui-0.9.0/lib/CMakeLists.txt
===================================================================
--- ktp-text-ui-0.9.0.orig/lib/CMakeLists.txt
+++ ktp-text-ui-0.9.0/lib/CMakeLists.txt
@@ -32,6 +32,8 @@ kde4_add_ui_files(ktpchat_SRCS ${ktpchat
 
 kde4_add_library(ktpchat SHARED ${ktpchat_SRCS})
 
+set_target_properties(ktpchat PROPERTIES SOVERSION ${KTP_TEXT_UI_VERSION} VERSION ${KTP_TEXT_UI_VERSION})
+
 target_link_libraries(ktpchat
     ${KDE4_KDECORE_LIBS}
     ${KDE4_KIO_LIBS}
